indico/cli/i18n.py

```diff
@@ -335,6 +335,68 @@ def remove_empty_pot_files(path: Path, python=False, javascript=False, react=Fal
         _get_messages_react_pot(path).unlink()
 
 
+# Merge after generating all the .pot files
+def merge_pot_files(output_file, *input_files):
```

Comment:
I think this is one of the places where type annotations are really useful so you get autocompletion for all the pathlib stuff (assuming all the args are Paths)